paper-dropdown-menu.html

```diff
@@ -7,178 +8,319 @@
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-menu-button/paper-menu-button.html">
+<link rel="import" href="../paper-ripple/paper-ripple.html">
+<link rel="import" href="../iron-a11y-keys-behavior/iron-a11y-keys-behavior.html">
+<link rel="import" href="../iron-behaviors/iron-button-state.html">
+<link rel="import" href="../iron-icons/iron-icons.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../iron-selector/iron-selectable.html">
 
-`paper-dropdown-menu` works together with `paper-dropdown` and `core-menu` to
-implement a drop-down menu. The currently selected item is displayed in the
-control. If no item is selected, the `label` is displayed instead.
+<!--
+`paper-dropdown-menu` is similar to a native browser select element.
+`paper-dropdown-menu` works with selectable content. The currently selected
+item is displayed in the control. If no item is selected, the `label` is
+displayed instead.
 
-The child element with the class `dropdown` will be used as the drop-down
-menu. It should be a `paper-dropdown` or other overlay element. You should
-also provide a `core-selector` or other selector element, such as `core-menu`,
-in the drop-down. You should apply the class `menu` to the selector element.
+The child element with the class `dropdown-content` will be used as the dropdown
+menu. It could be a `paper-menu` or element that triggers `iron-activate` when
+selecting its children.
 
 Example:
 
     <paper-dropdown-menu label="Your favorite pastry">
-        <paper-dropdown class="dropdown">
-            <core-menu class="menu">
-                <paper-item>Croissant</paper-item>
-                <paper-item>Donut</paper-item>
-                <paper-item>Financier</paper-item>
-                <paper-item>Madeleine</paper-item>
-            </core-menu>
-        </paper-dropdown>
+      <paper-menu class="dropdown-content">
+        <paper-item>Croissant</paper-item>
+        <paper-item>Donut</paper-item>
+        <paper-item>Financier</paper-item>
+        <paper-item>Madeleine</paper-item>
+      </paper-menu>
     </paper-dropdown-menu>
 
-This example renders a drop-down menu with 4 options.
+This example renders a dropdown menu with 4 options.
+
+### Styling
+
+The following custom properties and mixins are also available for styling:
+
+Custom property | Description | Default
+----------------|-------------|----------
+`--paper-dropdown-menu` | A mixin that is applied to the element host | `{}`
+`--paper-dropdown-menu-disabled` | A mixin that is applied to the element host when disabled | `{}`
+`--paper-dropdown-menu-ripple` | A mixin that is applied to the internal ripple | `{}`
+`--paper-dropdown-menu-button` | A mixin that is applied to the internal menu button | `{}`
+`--paper-dropdown-menu-input` | A mixin that is applied to the internal paper input | `{}`
+`--paper-dropdown-menu-icon` | A mixin that is applied to the internal icon | `{}`
 
```

Comment:
Kind of thinking about adding a bonus line, like: "You can also use any of the Polymer.PaperInputContainer styles to style the internal input". I think that's also true, isn't it?